test/k8s-integration/main.go

```diff
@@ -216,10 +216,16 @@ func handle() error {
 		if err != nil {
 			return fmt.Errorf("failed to build Kubernetes: %v", err)
 		}
+		err = buildKubernetes(testDir, "ginkgo")
```

Comment:
related to this pr? we use kubetest now

Reply:
gingko needs to be built for kubetest to work.

Reply:
can you add a comment here. that's not immediately obvious and could get easily be accidentally removed as "dead code" later